datasets/lt/magnitsky_amendments/crawler.py

```diff
@@ -62,8 +61,9 @@ def crawl(context: Context):
         # Create a Sanction entity
         sanction = h.make_sanction(context, person)
         sanction.add("reason", entry.get("priezastis"), lang="lit")
-        # sanction.add("startDate", h.parse_date("2018-01-01", DATE_FORMATS))
-        sanction.add("endDate", h.parse_date(entry.get("uzdraustaIki"), DATE_FORMATS))
+        for date in h.multi_split(entry.get("priezastis"), ["galiojanti nuo "]):
+            h.apply_date(sanction, "startDate", date)
```

Comment:
is this the start date of the law or the ban? it looks like new people have recently been added, but they still get the 2018 date which makes me think it's not really right thing thing for an individual's sanction